jsx/dashboard/config.jsx

```diff
@@ -32,29 +32,54 @@ class config extends React.Component {
   }
 
   save () {
+    let error = false
     let payload = this.state.config
     payload.width = parseInt(payload.width)
     payload.height = parseInt(payload.height)
     payload.column = parseInt(payload.column)
     payload.row = parseInt(payload.row)
-    payload.width = parseInt(payload.width)
-    this.props.updateDashboard(payload)
-    this.setState({updated: false})
+    const fieldsToCheck = ['width', 'height', 'column', 'row']
+    for (let prop of fieldsToCheck) {
+      if (payload[prop] <= 0) {
+        console.log('invalid input')
+        error = true
+      }
+    }
+    if (!error) {
+      this.props.updateDashboard(payload)
+      this.setState({ updated: false })
```

Comment:
With this we’ll have two different styles of validations. Is that normal ? I am ok with as long as we are all on same page and we know when to use which one . @Ranthalion comments?

Reply:
In fact this part is not really meant to be triggered because of the validation on the fields. If we trigger this error it means we have really messy data.

Reply:
Updating to use Formik requires a pretty big change, so I'm OK with both patterns being in use.

Reply:
And to be honest i don't think Formik fits in this part at all.